SPECS/application-gateway-kubernetes-ingress/application-gateway-kubernetes-ingress.spec

```
@@ -20,12 +20,10 @@ Source0:        https://github.com/Azure/application-gateway-kubernetes-ingress/
 #           --mtime="2021-04-26 00:00Z" \
 #           --owner=0 --group=0 --numeric-owner \
 #           --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \
-#           -cf %%{name}-%%{version}-vendor.tar.gz vendor
+#           -cf %%{name}-%%{version}-%%{release}-vendor.tar.gz vendor
 #
 Source1:        %{name}-%{version}-vendor.tar.gz
```

```suggestion
Source1:        %{name}-%{version}-govendor-v1.tar.gz
```
This should be changed to match the new format of `%{name}-%{version}-govendor-v1.tar.gz` (or v2, v3, etc). Although this is an updated tarball for the same version, I'm inclined to go with `v1` since we don't have a `v1` right now, and it'll help keep things consistent for any future work.

---

thanks for the guidance, agreed